reactioncommerce/reaction
# feat: using account-js for auth

BREAKING CHANGE: hydra and idenity are no longer shipped by default

Signed-off-by: Akarshit Wal <akarshitwal@gmail.com>

## Testing
1. Pull in the storefront branch corresponding to accounts-js
2. Use the reaction-dev-platform to start the containers
3. Try to login on the example-storefront and make sure the new API for auth is working.
